src/components/DatePicker/DatePicker.st.css

```diff
@@ -534,12 +545,19 @@
 /*    color: value(CurrentDisabledDayColor);*/
 /*}*/
 
+/*.overrideStyleParams .calendar::day:focus::after {*/
+/*    border-radius: value(CurrentSelectedDayBorderRadius);*/
+/*}*/
+
 /*.overrideStyleParams .calendar::day:singleDay:not(:outside) .dayWrapper {*/
 /*    background-color: value(CurrentSelectedDayBGColor);*/
 /*}*/
 /*.overrideStyleParams .calendar::day:singleDay:not(:outside):hover .dayWrapper {*/
 /*    background-color: applyOpacity(value(CurrentSelectedDayBGColor), 0.7);*/
 /*}*/
+/*.overrideStyleParams .dayWrapper {*/
+/*    border-radius: value(CurrentSelectedDayBorderRadius);*/
+/*}*/
```

Comment:
Do we want to leave the commented code?
It's a setup for the future task?

Reply:
@nitayneeman, @jonathanadler :
I'm answering here for both comments :)
I tried to change it to the syntax of overrideStyleParams but there were issues so I put it in a comment for later.
I can remove it but I think it will be easier to use it later.